doc/classes/Crypto.xml

```diff
@@ -81,7 +81,7 @@
 			<argument index="1" name="received" type="PackedByteArray">
 			</argument>
 			<description>
-				Compares two [PackedByteArray]s for equality without leaking timing information in order to prevent timing attacks.
```

Comment:
I think this makes it worse. "in order to" is more explicit than "to" in this context.